rpb/riak_dt/riak_dt.pb.go

```diff
@@ -75,6 +81,7 @@ func (x *MapField_MapFieldType) UnmarshalJSON(data []byte) error {
 	*x = MapField_MapFieldType(value)
 	return nil
 }
+func (MapField_MapFieldType) EnumDescriptor() ([]byte, []int) { return fileDescriptor0, []int{0, 0} }
```

Comment:
I just did the following and do not see all of these changes:

```sh
cd riak_pb
git fetch origin
git checkout 2.1.3.0
cd ..
make protogen
```

See PR #69. Let me know how you'd like to proceed.